assets/js/barChart.js

```diff
@@ -70,12 +74,27 @@ class BarChart {
 
         this.rects = bars.append('rect')
             .attr('y', function(d) { return yScale(d.name); })
-            .attr('height', yScale.rangeBand());
+            .attr('height', yScale.rangeBand())
+            // Add a tooltip to the bar.
+            .on("mouseover", function (d) { tooltip.style("display", "inline-block"); })
+            .on("mousemove", function (d) {
+                tooltip
+                    .style("left", d3.event.pageX - 50 + "px")
+                    .style("top", d3.event.pageY - 70 + "px")
+                    .style("display", "inline-block")
+                    .html((d.name.replace('<', '&lt;') + ' (' + d.val + ')'));
```

Comment:
Thanks @jdaguil  -- and I see the reference in the [D3 3.x API](https://github.com/d3/d3-3.x-api-reference/blob/master/Selections.md#html).